opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/export/__init__.py

```diff
@@ -137,20 +143,66 @@ class BatchLogRecordProcessor(LogRecordProcessor):
     """This is an implementation of LogRecordProcessor which creates batches of
     received logs in the export-friendly LogData representation and
     send to the configured LogExporter, as soon as they are emitted.
+
+    `BatchLogRecordProcessor` is configurable with the following environment
+    variables which correspond to constructor parameters:
+
+    - :envvar:`OTEL_BLRP_SCHEDULE_DELAY`
+    - :envvar:`OTEL_BLRP_MAX_QUEUE_SIZE`
+    - :envvar:`OTEL_BLRP_MAX_EXPORT_BATCH_SIZE`
+    - :envvar:`OTEL_BLRP_EXPORT_TIMEOUT`
     """
 
     def __init__(
         self,
         exporter: LogExporter,
-        schedule_delay_millis: int = 5000,
-        max_export_batch_size: int = 512,
-        export_timeout_millis: int = 30000,
+        max_queue_size: int = None,
+        schedule_delay_millis: int = None,
+        max_export_batch_size: int = None,
+        export_timeout_millis: int = None,
```

Comment:
Figured the priority should be parity with ordering of BSP, but I can make this change.

Reply:
I didn't know that was the order for BSP; You can keep it consistent with the BSP but highlight it in the CHANGELOG. We are technically allowed to introduce breaking changes since logs is still experimental.